package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vote_tracker",
+  "version": "1.0.0",
+  "description": "Have a basic semi functional vote tracker website. Only thing missing is the, well, vote tracking. Have it at a point where no 2 cats will appear at the same time.",
+  "main": "chart.min.js",
```

Comment:
This line that starts with 'main:' should point to the js file that you want node to start up (the main entry point to your application). In your apps case, it should be 'main: server.js'.